kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,104 @@ This sample demonstrates how to install the Traefik ingress controller to provid
-#### Install a host-routing Ingress
+### 2. Web request routing 
+#### Host based routing 
+This sample demonstrates how to access application on two WebLogic domains using host based routing. Install a host-routing Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute)
 ```
 $ kubectl create -f samples/host-routing.yaml
+ingressroute.traefik.containo.us/traefik-hostrouting-1 created
+ingressroute.traefik.containo.us/traefik-hostrouting-2 created
 ```
-Now you can send requests to different WLS domains with the unique entry point of Traefik with different hostnames.
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different hostnames as defined in route section of `host-routing.yaml`.
 ```
-$ curl -H 'host: domain1.org' http://${HOSTNAME}:30305/testwebapp/
-$ curl -H 'host: domain2.org' http://${HOSTNAME}:30305/testwebapp/
+# Get the ingress controller web port
+$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')
+$ curl -H 'host: domain1.org' http://${HOSTNAME}:${LB_PORT}/testwebapp/
+$ curl -H 'host: domain2.org' http://${HOSTNAME}:${LB_PORT}/testwebapp/
 ```
-#### Install a path-routing Ingress
+
+#### Path based routing  
+This sample demonstrates how to access application on two WebLogic domains using path based routing. Install a path-routing Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute) and [Middleware](https://docs.traefik.io/middlewares/overview/)
+
 ```
 $ kubectl create -f samples/path-routing.yaml
+ingressroute.traefik.containo.us/traefik-pathrouting-1 created
+middleware.traefik.containo.us/middleware-domain1 created
+ingressroute.traefik.containo.us/traefik-pathrouting-2 created
+middleware.traefik.containo.us/middleware-domain2 created
 ```
-Now you can send requests to different WLS domains with the unique entry point of Traefik with different paths.
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different paths as defined in route section of `path-routing.yaml`.
 ```
-$ curl http://${HOSTNAME}:30305/domain1/
-$ curl http://${HOSTNAME}:30305/domain2/
+# Get the ingress controller web port
+$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')
+$ curl http://${HOSTNAME}:${LB_PORT}/domain1/
+$ curl http://${HOSTNAME}:${LB_PORT}/domain2/
 ```
-#### Install a TLS-enabled Ingress
-This sample demonstrates accessing the two WLS domains using an HTTPS endpoint and the WLS domains are protected by different TLS certificates.
+#### Host based secured routing 
+This sample demonstrates how to access application on two WebLogic domains using a HTTPS endpoint. Install a TLS-enabled Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute)
```

Comment:
access application -> access an application
a HTTPS -> an HTTPS
Install a TLS-enabled Traefik [IngressRoute](https://docs.traefik.io/routing/providers/kubernetes-crd/#kind-ingressroute).  (period at the end of the sentence)

Reply:
done